src/components/site/SiteStatus.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import SiteStatus from "./SiteStatus";
+
+describe("Text Input", () => {
```

Comment:
please remove the describe, in this case, reasoning the same as above.

Reply:
done